Production/govuk_ios/ViewControllers/SearchViewController.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import UIComponents
+
+class SearchViewController: BaseViewController,
+                            TrackableScreen {
+    private let viewModel: SearchViewModel
+
+    private lazy var searchBar: UISearchBar = {
+        let localSearchBar = UISearchBar()
+
+        localSearchBar.searchTextField.backgroundColor = UIColor.govUK.fills.surfaceSearchBox
+        localSearchBar.enablesReturnKeyAutomatically = false
+        localSearchBar.translatesAutoresizingMaskIntoConstraints = false
+        localSearchBar.placeholder = "Search"
```

Comment:
There's now a localizable strings file we could add this to ready for when we add Welsh translation? Could do same for all other new string literals in this PR?

Reply:
Good point 👍 Transferred user facing strings to locale file